homeassistant/components/peco/sensor.py

```diff
@@ -108,6 +103,6 @@ def __init__(
         self.entity_description = description
 
     @property
-    def native_value(self) -> float:
+    def native_value(self) -> int:
```

Comment:
Why not just return a float?

Reply:
Because it wouldn't make sense for customers to see that there is "486.0" customers out instead of "486"